string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,12 @@
+/* exported capitalize */
+// capitalize entire word
+// separately save the first letter of word
+// separately make rest of word lower case
+// add both words together
+function capitalize(word) {
+  word = word.toUpperCase();
+  var wordFirst = word[0];
```

Comment:
We could have apply `toUppercase` to `wordFirst` instead of having whole word capitalized